install/kubernetes/helm/istio/charts/galley/templates/deployment.yaml

```diff
@@ -42,16 +42,16 @@ spec:
 {{- end }}
           imagePullPolicy: {{ .Values.global.imagePullPolicy }}
           ports:
-          - containerPort: 443
+          - containerPort: {{ .Values.validationPort }}
           - containerPort: {{ .Values.global.monitoringPort }}
           - containerPort: 9901
           command:
           - /usr/local/bin/galley
           - server
           - --meshConfigFile=/etc/mesh-config/mesh
           - --livenessProbeInterval=1s
-          - --livenessProbePath=/healthliveness
-          - --readinessProbePath=/healthready
+          - --livenessProbePath=/tmp/healthliveness
```

Comment:
Why ?

Reply:
nonroot user does not have permission to create file in `/`, that is why change it to `/tmp/xxx`. what is the problem for `/tmp`?